.github/workflows/ci_zig.yml

```diff
@@ -98,8 +98,11 @@ jobs:
           sudo rm /usr/lib/llvm-18/bin/llvm-config
 
       - name: build repro executables
-        run: |
-          zig build -Dfuzz -Dsystem-afl=false
+        uses: ./.github/actions/flaky-retry
+        with:
+          command: 'zig build -Dfuzz -Dsystem-afl=false'
+          error_string_contains: 'EndOfStream'
+          retry_count: 3
```

Comment:
Why is zig build flaky?

Reply:
Oh woah this change was not intended. Not sure how this snuck in; probably junk from bisecting or something.

Reply:
Heh, somehow accidentally amending a merge commit from master resulted in this. Unclear why. But undone now.

Reply:
Issue for flaky EndOfStream error when getting dependencies: https://github.com/ziglang/zig/issues/21316